tests/build.rs

```diff
@@ -54,7 +54,11 @@ fn clone_build_and_clean() -> anyhow::Result<()> {
     let file = assert_fs::NamedTempFile::new(constants::LLVM_LOCK_FILE)?;
     let path = file.parent().unwrap();
     cmd.current_dir(path);
-    file.write_str(&*format!("url = \"{}\"", constants::ERA_LLVM_REPO_URL))?;
+    file.write_str(&*format!(
```

Comment:
Please use a TOML serializer with the `Lock` structure.

Reply:
Very good point. Updated and refactored the code to use TOML serialization.